app/views/govuk_publishing_components/components/_cookie_banner.html.erb

```diff
@@ -8,7 +10,7 @@
       <div class=" govuk-grid-column-two-thirds">
         <div class="gem-c-cookie-banner__message">
           <span class="govuk-heading-m">Tell us whether you accept cookies</span>
```

Comment:
While we're at it, is it worth having a custom option this heading text as well?

Reply:
Hm, I did consider it but thought I'd keep this change as small as possible - at the moment we don't have a use case for changing the title.

Reply:
I was going to ask the same about the button text but concluded no use case, no need.

Reply:
Fair enough.